specs/0101-terminology-presentation.md

```diff
@@ -0,0 +1,41 @@
+- Feature Name: Terminology Presentation
+- Created: 2020-03-24
+- Associated Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1624557
+
+# Summary
+
+Terminology is a group of specialized words, compound words or multi-word expressions (terms) relating to a particular field. This feature adds the ability to store such terms in Pontoon, identify them in original strings and present them in the translation workbench.
+
+# Motivation
+
+- **Improving translation consistency**: it’s hard to expect all authors to always use the same translations for the same words and phrases without any help from the tool.
+- **Speeding up translation process**: localizers need to manually search for existing translations of terms found in the strings they are translating.
+- **Fixing brand translations**: trademarks present a special case for localization as they have legal and semantic significance.
+
+# Out of scope
+
+- **Terminology creation**: the ability to manually add, remove and review terms.
+- **Terminology translation**: the ability to manually add, remove and review term translations.
+- **VCS intergration**: the ability to sync terminology with VCS.
+- **Checks**: the ability to check if translations include term translations.
+
+# Feature explanation
+
+As the first step, we need the ability to store terms in the database. Data model should allow for importing terms stored in the TBX format and the [initial Mozilla terms list](https://docs.google.com/spreadsheets/d/1MAPD8WBnstR6pwKbNEDKOpw5CTPnl3qAobDgomdmtdY/edit?ts=5e79126c#gid=1146590716).
+
+The list of fields should include at least the following fields:
+- term
+- part of speech
+- definition
+- usage
+- translations
```

Comment:
One or many translations? Jeff said that one-to-one is good enough, but we should be precise about that.

Reply:
I was thinking of Term Translations as a separate table. I'll clarify that.